nekoyume/Assets/_Scripts/Helper/Util.cs

```diff
@@ -45,5 +48,42 @@ public static string GetBlockToTime(int block)
 
             return sb.ToString();
         }
+
+        public static Order GetOrder(Guid orderId)
+        {
+            var address = Order.DeriveAddress(orderId);
+            var state = Game.Game.instance.Agent.GetState(address);
+            if (state is Dictionary dictionary)
+            {
+                return OrderFactory.Deserialize(dictionary);
+            }
+
+            return null;
+        }
+
+        public static ItemBase GetItemBaseByOrderId(Guid orderId)
+        {
+            var order = GetOrder(orderId);
+            return GetItemBaseByTradableId(order.TradableId);
+        }
+
+        public static ItemBase GetItemBaseByTradableId(Guid tradableId)
+        {
+            var address = Addresses.GetItemAddress(tradableId);
+            var state = Game.Game.instance.Agent.GetState(address);
+            if (state is Dictionary dictionary)
+            {
+                return ItemFactory.Deserialize(dictionary);
+            }
+
+            return null;
+        }
+
+        public static ItemBase GetItemBaseByItemId(int itemId)
```

Comment:
```suggestion
        public static ItemBase CreateItemBaseByItemId(int itemId)
```

How about you move this method to `ItemFactory`.